src/SfxWeb/src/app/views/cluster/commands/commands.component.ts

```diff
@@ -35,12 +35,84 @@ export class CommandsComponent extends BaseControllerDirective {
     this.setUpCommands();
   }
 
-
   protected setUpCommands(): void {
 
     const healthReport = CommandFactory.GenSendHealthReport("Cluster");
     this.commands.push(healthReport);
-    
+
+    const connectHelp = 'https://docs.microsoft.com/powershell/module/servicefabric/connect-servicefabriccluster';
+    const connectionEndpoint = new PowershellCommandParameter("ConnectionEndpoint", CommandParamTypes.string, { required: true, defaultValue: document.location.hostname + ":19000"});
+    const storeLocation = new PowershellCommandParameter("StoreLocation", CommandParamTypes.enum, { required: true, defaultValue: "CurrentUser", options: ["CurrentUser", "LocalMachine"]});
+
+    let authType = "-X509Credential ";
+    if (this.data.clusterManifest.isAADEnabled) {
+      authType = "-AzureActiveDirectory ";
+    }
+
+    let certificateProperties = this.data.clusterManifest.certificatesInfo;
+    let clientThumbprints = certificateProperties
+      .filter(x => x.name === "ClientCertificate" && x.x509FindType !== "FindBySubjectName")
+      .map(x => { return x.x509FindValue });
+    let clientCommonNames = certificateProperties
+      .filter(x => x.name === "ClientCertificate" && x.x509FindType === "FindBySubjectName")
+      .map(x => { return x.x509FindValue });
+    let clusterThumbprints = certificateProperties
+      .filter(x => x.name === "ClusterCertificate" && x.x509FindType !== "FindBySubjectName")
+      .map(x => { return x.x509FindValue });
+    let clusterCommonNames = certificateProperties
+      .filter(x => x.name === "ClusterCertificate" && x.x509FindType === "FindBySubjectName")
+      .map(x => { return x.x509FindValue });
+    let serverThumbprints = certificateProperties
+      .filter(x => x.name === "ServerCertificate" && x.x509FindType !== "FindBySubjectName")
+      .map(x => { return x.x509FindValue });
+    let serverCommonNames = certificateProperties
+      .filter(x => x.name === "ServerCertificate" && x.x509FindType === "FindBySubjectName")
+      .map(x => { return x.x509FindValue });
+
+    const serverThumbprint = new PowershellCommandParameter("ServerCertThumbprint", CommandParamTypes.enum, {
+      required: true,
+      options: [...new Set(serverThumbprints)],
```

Comment:
There could be a secondary Server certificates listed, but only one of them would work. Since the `ServerCertificate` parameters accept a string array, put all of the certificates in as one value